public/app/features/plugins/admin/components/PluginDetailsHeader.tsx

```diff
@@ -26,75 +26,79 @@ export function PluginDetailsHeader({ plugin, currentUrl, parentUrl }: Props): R
   const version = plugin.installedVersion || latestCompatibleVersion?.version;
 
   return (
-    <div className={styles.headerContainer}>
-      <PluginLogo
-        alt={`${plugin.name} logo`}
-        src={plugin.info.logos.small}
-        className={css`
-          object-fit: contain;
-          width: 100%;
-          height: 68px;
-          max-width: 68px;
-        `}
-      />
-
-      <div className={styles.headerWrapper}>
-        {/* Title & navigation */}
-        <nav className={styles.breadcrumb} aria-label="Breadcrumb">
-          <ol>
-            <li>
-              <a className={styles.textUnderline} href={parentUrl}>
-                Plugins
-              </a>
-            </li>
-            <li>
-              <a href={currentUrl} aria-current="page">
-                {plugin.name}
-              </a>
-            </li>
-          </ol>
-        </nav>
-
-        <div className={styles.headerInformationRow}>
-          {/* Org name */}
-          <span>{plugin.orgName}</span>
-
-          {/* Links */}
-          {plugin.details?.links.map((link: any) => (
-            <a key={link.name} href={link.url}>
-              {link.name}
-            </a>
-          ))}
-
-          {/* Downloads */}
-          {plugin.downloads > 0 && (
-            <span>
-              <Icon name="cloud-download" />
-              {` ${new Intl.NumberFormat().format(plugin.downloads)}`}{' '}
-            </span>
-          )}
-
-          {/* Version */}
-          {Boolean(version) && <span>{version}</span>}
-
-          {/* Signature information */}
-          <PluginDetailsHeaderSignature plugin={plugin} />
-
-          {plugin.isDisabled && <PluginDisabledBadge error={plugin.error!} />}
+    <div>
```

Comment:
this extra div is a bit hacky, it's to work around an issue with flexbox and that page-container has a flex-basis: 100% and so if parent is a flexbox with layout column it will take a lot height